Gordon360/Controllers/MembershipsController.cs

```diff
@@ -49,11 +32,11 @@ public async Task<ActionResult<IEnumerable<MembershipViewModel>>> GetAsync()
         /// <param name="sessionCode">Optional code of session to get for</param>
         /// <returns>IHttpActionResult</returns>
         [HttpGet]
-        [Route("activity/{activityCode}")]
+        [Route("activities/{activityCode}/sessions/{sessionCode}")]
         [StateYourBusiness(operation = Operation.READ_PARTIAL, resource = Resource.MEMBERSHIP_BY_ACTIVITY)]
-        public async Task<ActionResult<IEnumerable<MembershipViewModel>>> GetMembershipsForActivityAsync(string activityCode, string? sessionCode)
+        public ActionResult<IEnumerable<MembershipViewModel>> GetMembershipsForActivity(string activityCode, string? sessionCode)
```

Comment:
```suggestion
        public ActionResult<IEnumerable<MembershipViewModel>> GetMembershipsForActivity(string activityCode, string sessionCode)
```

Reply:
Yea, reverting this change as we discussed and planning to address the /activities/:id/sessions/:id structure later.